src/voice-broadcast/models/VoiceBroadcastRecording.ts

```diff
@@ -333,10 +334,27 @@ export class VoiceBroadcastRecording
      * It sets the connection error state and stops the recorder.
      */
     private async onConnectionError(): Promise<void> {
+        this.playConnectionErrorAudioNotification().catch(() => {
+            // Error logged in playConnectionErrorAudioNotification().
+        });
         await this.stopRecorder(false);
         this.setState("connection_error");
     }
 
+    private async playConnectionErrorAudioNotification(): Promise<void> {
+        if (localNotificationsAreSilenced(this.client)) {
+            return;
+        }
+
+        const audioElement = document.querySelector<HTMLAudioElement>("audio#errorAudio");
```

Comment:
How does this element get into the the document?

Reply:
Very good question :laughing: I forgot to link the Element Web PR. Also a comment would probably be a good idea…